UnitsNet.Tests/CustomCode/DynamicViscosityTests.cs

```diff
@@ -14,6 +14,7 @@ public class DynamicViscosityTests : DynamicViscosityTestsBase
         protected override double NewtonSecondsPerMeterSquaredInOneNewtonSecondPerMeterSquared => 1;
         protected override double PascalSecondsInOneNewtonSecondPerMeterSquared => 1;
         protected override double PoiseInOneNewtonSecondPerMeterSquared => 10;
+        protected override double PoundsFootSecondInOneNewtonSecondPerMeterSquared => 0.671968994813;
```

Comment:
Verified. 
https://www.convertunits.com/from/pound/foot-second/to/newton+second/square+meter